src/pages/projects/AllProjects.tsx

```diff
@@ -36,9 +42,20 @@ export default function AllProjects({
     pv,
   })
 
-  const { data: searchPages, isLoading: isLoadingSearch } =
+  const { data: sepanaSearchResults, isLoading: isLoadingSepanaSearch } =
+    useSepanaProjectsSearch(searchText)
+
+  const { data: graphSearchResults, isLoading: isLoadingGraphSearch } =
     useProjectsSearch(searchText)
 
+  const sepanaEnabled = featureFlagEnabled(FEATURE_FLAGS.SEPANA_SEARCH)
+
+  const searchResults = sepanaEnabled ? sepanaSearchResults : graphSearchResults
+
+  const isLoadingSearch = sepanaEnabled
```

Comment:
**suggestion** it looks like this'll do 2 requests even though the feature flag means we only need data from 1 of them. Is this intentional? (I can see how it might be, cos then we get some in-prod testing on the api requests and queries, without the user seeing them).

If it's not, WDYT about passing an `enabled` flag to each of useSepanaProjectsSearch and useProjectsSearch, and enabling them depending on feature flag state?

Reply:
yeah, currently making 2 queries. i could add a clause in each hook like `if (x) don't fetch`, but i think it'd be useless outside of this use case

i'll see if i can find a conditional that makes sense to avoid double query

Reply:
useQuery has an `enabled` flag which we could use:

https://github.com/jbx-protocol/juice-interface/blob/71bef62f51d00ccb61573d4e7a0ce0c1aeefc6c4/src/hooks/veNft/VeNftContractForProject.ts#L19

Reply:
ohh, perfect